package.json

```diff
@@ -21,7 +21,7 @@
   },
   "size-limit": [
     {
-      "limit": "1.8 KB",
+      "limit": "1.875 KB",
```

Comment:
annoying we need to increase this, but the benefit I believe is a more understandable solution and easier code to reason about